x-pack/plugins/cloud_security_posture/public/pages/configurations/latest_findings/latest_findings_table.tsx

```diff
@@ -63,11 +63,12 @@ const customCellRenderer = (rows: DataTableRecord[]) => ({
 
     return <CspEvaluationBadge type={finding?.result?.evaluation} />;
   },
-  'data_stream.dataset': ({ rowIndex }: EuiDataGridCellValueElementProps) => {
+  'observer.vendor': ({ rowIndex }: EuiDataGridCellValueElementProps) => {
     const finding = getCspFinding(rows[rowIndex].raw._source);
-    const source = getDatasetDisplayName(finding?.data_stream?.dataset);
+    const vendor =
+      finding?.observer?.vendor || getDatasetDisplayName(finding?.data_stream?.dataset);
```

Comment:
if we move all this logic into `getDatasetDisplayName`, i think it can be simplified to just 
```
displayName = getDatasetDisplayName(...);

return <>{displayName || ''}</>;
```